clustercontroller-core/src/test/java/com/yahoo/vespa/clustercontroller/core/NodeStateChangeCheckerTest.java

```diff
@@ -753,4 +699,62 @@ private List<ConfiguredNode> createNodes(int count) {
             nodes.add(new ConfiguredNode(i, false));
         return nodes;
     }
+
+    private StorDistributionConfig createDistributionConfig(int nodes) {
+        var configBuilder = new StorDistributionConfig.Builder()
+                .ready_copies(requiredRedundancy)
+                .redundancy(requiredRedundancy)
+                .initial_redundancy(requiredRedundancy);
+
+        var groupBuilder = new StorDistributionConfig.Group.Builder()
+                                    .index("invalid")
+                                    .name("invalid")
+                                    .capacity(nodes);
+        int nodeIndex = 0;
+        for (int j = 0; j < nodes; ++j, ++nodeIndex) {
```

Comment:
For this particular loop I think it would make sense to collapse `nodeIndex` and `j` into a single `nodeIndex` loop variable